crates/net/eth-wire/src/types/message.rs

```diff
@@ -153,6 +155,7 @@ pub enum EthMessage {
     NewBlock(Box<NewBlock>),
     Transactions(Transactions),
     NewPooledTransactionHashes(NewPooledTransactionHashes),
+    NewPooledTransactionHashes68(NewPooledTransactionHashes68),
```

Comment:
this is a bit cursed -.- that the variant now is an enum

Reply:
Yeah, I think given we don't have support for multiple capabilities, which also has the need for new message types and encodings, it's up in the air how we should do this properly. But yeah, I agree this doesn't seem like the best way to do it

Reply:
I agree it's too ugly and immature. 

I adopted the convention from `geth`. (They uses the name like `NewPooledTransactionHashesPacket66` and `NewPooledTransactionHashesPacket68`.) And I tried to minimize changes. This code is the result of this approach.

I'll think more what's better way.

Reply:
I actually like that we do version-specific encoding on the ethstream level.

so, after some thinking this change seems fine to me, because we already have outdated variants like GetNodeData etc.
and because the stream does not emit them if they don't match the version this is ok.

Reply:
I'd like to rename the existing variant to `NewPooledTransactionHashes66` then

Reply:
> I actually like that we do version-specific decoding on the ethstream level.

agreed and I had tried this way too. I tried to make ethstream as generic depending on eth version. I thought this way is clean but I couldn't implement it with Rust.

Here is my trial (but failed): https://github.com/jinsankim/reth/commit/675d7ec25f6b15f9977f7d59c5c59381bee853a9

Reply:
> I'd like to rename the existing variant to NewPooledTransactionHashes66 then

That's my first implementation but I've reverted it. I'll think again and share why I reverted it. Or will follow up the idea.

Reply:
I'm learning for `ActiveSession`, `PeerMessage::PooledTransactions` and so on. The PR might have a bug.

Reply:
After learning code more, I could remember why I choose to use `NewPooledTransactionHashes` than `NewPooledTransactionHashes66`. It's because `NewPooledTransactionHashes` is used internally as well for `PeerMessage::PooledTransactions`, `NetworkHandleMessage::SendPooledTransactionHashes` and `NetworkTransactionEvent::IncomingPooledTransactionHashes`. 

Now I've realized that we should care of these as well. maybe need more `66`, `68` convention and branches. T_T I'm working on it.

Reply:
> I'd like to rename the existing variant to NewPooledTransactionHashes66 then

follow it up with a94b1da162f48db190c321fa8068b457bc439ce2. I've changed this PR into Draft because the new change is little big. I need to add more tests and review myself first.

Reply:
ready for review.